app/build.gradle.kts

```
@@ -22,8 +28,9 @@ android {
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
 
-        //TODO add base url
-        buildConfigField("String", "BASE_URL", "\"https://google.com\"")
+        buildConfigField("String", "BASE_URL", "\"https://api.themoviedb.org\"")
```

it would be better to add version in base url

```suggestion
        buildConfigField("String", "BASE_URL", "\"https://api.themoviedb.org\3"")
```

---

fixed